OpenSim/Simulation/SimbodyEngine/Coordinate.h

```diff
@@ -330,7 +330,7 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Coordinate, ModelComponent);
     SimTK::ReferencePtr<const Joint> _joint;
 
     /* User set MotionType from versions of OpenSim that predate 4.0 */
-    MotionType _userSpecifiedMotionTypePriorTo40;
+    MotionType _userSpecifiedMotionTypePriorTo40{ Undefined };
```

Comment:
@aseth1 is this related to the undefined motion type reported by Colin earlier? does it fix it, or try to address it?

Reply:
I thought it might have been but it was not the cause. Since we have `Undefined` as a value, I think it is cleaner to initialize with it.